client/src/components/BookDetail/BookDetail.jsx

```diff
@@ -138,7 +138,7 @@ export default function BookDetail(props) {
                     {/* <input className="rating-type" type="radio" value="1"/> */}
                     <input type="submit" value="Send" onClick={(e) => {e.preventDefault(); addReview()}}/>
                 </form>
-                <div className="ratings">{reviews.reverse().map(r => {
+                <div className="ratings">{reviews.slice().reverse().map(r => {
```

Comment:
Hey Nuha, what exactly are you testing here?

Reply:
hi! i was just doing a test commit to make sure I could create the PR properly--I'll use more descriptive commit names from now on.